composer.json

```diff
@@ -41,7 +41,6 @@
     },
     "suggest": {
         "ext-mbstring": "For handling non-UTF8 characters in cache signature.",
-        "ext-xml": "For better performance.",
```

Comment:
Well, until php 7.2 is out, and this project supports it, surely we should still recommend the xml extension for better performance?